nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/test/java/org/apache/nifi/controller/status/history/EmbeddedQuestDbRolloverHandlerTest.java

```diff
@@ -43,7 +42,10 @@
 import java.util.concurrent.TimeUnit;
 import java.util.function.Supplier;
 
-@Ignore("Buggy tests depend on time of day")
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@Disabled("Buggy tests depend on time of day")
```

Comment:
This is a good opportunity to remove this class based on the comments. This file can be deleted.